src/components/ProductImageUpload.tsx

```
@@ -0,0 +1,565 @@
+                    이미지를 드래그하여 업로드
+                  </p>
+                  <p 
+                    className="text-sm mb-4"
+                    style={{ color: 'var(--color-text-tertiary)' }}
+                  >
+                    또는 파일을 선택하세요
+                  </p>
+                  <Button
+                    type="button"
+                    style={{
+                      backgroundColor: 'var(--color-brand-primary)',
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)